app/templates/login.hbs

```diff
@@ -1,14 +1,5 @@
-{{#bs-form as |form|}}
-  {{#form.group as |group|}}
-    <label for="identification">Username</label>
-    {{input class='form-control' id='identification' placeholder='Enter Username' value=identification}}
-  {{/form.group}}
-  {{#form.group as |group|}}
-    <label for="password">Password</label>
-    {{input class='form-control' id='password' type='password' placeholder='Enter Password' value=password}}
-  {{/form.group}}
-  {{#if errorMessage}}
-    {{#bs-alert type='danger' dismissible=false}}Unable to authenticate: {{errorMessage}}{{/bs-alert}}
-  {{/if}}
-  {{#bs-button type='default' onClick=(action 'authenticate')}}Login{{/bs-button}}
-{{/bs-form}}
\ No newline at end of file
+<div class="jumbotron text-center">
+    <h2>Bespin: Bioinformatics Workflow Service</h2>
+    <p>Automated running of bioinformatic workflows.</p>
```

Comment:
Thanks for putting a login screen together and some introductory text. I know we'll have more to come here. May I suggest:

```
<h2>Bespin: On-demand genomic workflows</h2>
<p>Bespin is a service of the <a target="_blank" href="https://genome.duke.edu/cores/computing">Computational Solutions Shared Resources</a> at the <a target="_blank" href="https://www.genome.duke.edu">Duke Center for Genomic and Computational Biology</a></p>
```

Thoughts? @hlapp ?